### PR TITLE
fdavidcl's First Goodie

### PR DESCRIPTION
**Pull request by fdavidcl**
**What does your Instant Answer do?**  
This is the [Quick Start](https://duck.co/duckduckhack/goodie_quickstart) Goodie for my user. 

**What are some example queries that trigger this Instant Answer?**  
`duckduckhack fdavidcl`

**Where did you hear about DuckDuckHack? (For first time contributors)**  
I read about people contributing their own Instant Answers and decided to give it a try.

## Checklist
Please place an 'X' where appropriate.

```
- [X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [X] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Linux
        - [X] Firefox

    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```

### CODE REVIEW

**Comment by moollaza** [comment]
@fdavidcl thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

File: lib/DDG/Goodie/IsAwesome/fdavidcl.pm [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package DDG::Goodie::IsAwesome::fdavidcl;
+# ABSTRACT: Write an abstract here
```
Comment (moollaza) [inline lib/DDG/Goodie/IsAwesome/fdavidcl.pm]: Please remove these comments, they're not needed for us, they're just to help you along :)

File: lib/DDG/Goodie/IsAwesome/fdavidcl.pm [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+zci answer_type => "is_awesome_fdavidcl";
+zci is_cached   => 1;
+
+# Metadata.  See https://duck.co/duckduckhack/metadata for help in filling out this section.
+name "IsAwesome fdavidcl";
+description "Succinct explanation of what this instant answer does";
+primary_example_queries "first example query", "second example query";
+secondary_example_queries "optional -- demonstrate any additional triggers";
+# Uncomment and complete: https://duck.co/duckduckhack/metadata#category
```
Comment (moollaza) [inline lib/DDG/Goodie/IsAwesome/fdavidcl.pm]: Please remove these too

**Comment by moollaza** [comment]
@fdavidcl once the unneeded comments are removed, this should be good for merge!

**Comment by fdavidcl** [comment]
Thanks for the feedback! I've now mimicked the style of other Quick Start Goodies, should have paid more attention to that before.

**Comment by mintsoft** [comment]
Congratulations on your first goodie! @fdavidcl

**Comment by moollaza** [comment]
@fdavidcl nicely done!

Now that you've made your first Goodie, we really hope you'll continue to contribute and make more Instant Answers! Please checkout our [Ideas forum](https://duck.co/ideas) if you're unsure about what to build. There's also lots of Github Issues in our repos that need someone to tackle them!

**Comment by fdavidcl** [comment]
Thanks @mintsoft @moollaza!

**Comment by moollaza** [comment]
@fdavidcl congrats! It's live: https://duckduckgo.com/?q=duckduckhack+fdavidcl&t=ffcm&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**